client/components/notice/style.scss

```diff
@@ -11,7 +11,8 @@
 	}
 }
 
-.notice {
+ // Add an extra class name to specify the styles to avoid conflict with the WP-Admin notice styles.
+.notice.component-notice {
```

Comment:
Is there a reason we would rather add an extra class than simply renaming the class to something that won't collide?

Not just this specific component, but the general lack of class prefixes in the client components scare me. I'm thinking we should start adding a prefix, especially in cases like this where it's already known to have collisions.

Maybe `calypso-` for client components? I'm going for `a8c-` in `@automattic/components`.

`component-` is dangerously close to the `components-` prefix [used](https://github.com/WordPress/gutenberg/blob/b23709686bea6c64b33fd476437b0d5cfd68a382/packages/components/src/notice/style.scss#L1) in `@wordpress/components`. (I would've chosen a `wp-` prefix there but that ship has sailed 🤷)

Reply:
> Maybe calypso- for client components? I'm going for a8c- in @automattic/components.

Sounds like a good idea!

Reply:
> Is there a reason we would rather add an extra class than simply renaming the class to something that won't collide?

The main reason was not to cause an enormous refactor scope due to the renaming.

> but the general lack of class prefixes in the client components scare me. I'm thinking we should start adding a prefix, especially in cases like this where it's already known to have collisions.

@mirka Thanks for the review. I totally agree with it. They should have recognizable class names to resolve present and potential component conflict issues. It would be great to tackle this now if people like team Calypso, who often work on components, do not have much concern. 🙂